docs/UserGuide.md

```diff
@@ -66,8 +66,28 @@ This is a mockup of how the dashboard will look like (to be updated in v1.2):
 Allows the user to customise what to display on the dashboard. More details coming soon.
 
 
-## darren your work here (contacts)
-
+## Contacts
+
+### Viewing all contacts: `list`  
+Format: `list`  
+* Restores the contacts tab to its default view
+* Contacts are listed in alphabetical order of their names
+
+### Adding a contact: `add`
+Format: `type n/NAME [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]`  
+Examples:  
+* Adding a friend: `add friend n/NAME [p/PHONE] [e/EMAIL] [a/ADDRESS]`
+* Adding a client: `add client n/NAME [p/PHONE] [e/EMAIL] [a/ADDRESS]`
+### Deleting a contact: `delete`
```

Comment:
There are some issues with indentation here. Adding a newline should resolve it.
![Screenshot from 2021-09-29 22-14-37](https://user-images.githubusercontent.com/34470507/135286653-9da570ba-7662-47a9-a4b2-92f0fafec23f.png)

Reply:
Ok, pushed new to the PR, should be ok now